### PR TITLE
Feat/manager 15766 improuve can correct modal

### PR DESCRIPTION
**Pull request by jvaumoroovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? |no
| Tickets          | Fix # MANAFER-15766
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description
Now the `accelerate` operation is selected first. then the  `relaunch` operation and finally the `cancel`
Also move the `loading` in the .finally to be sure is called a the real end of the promise

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js [hunk 1]
```diff
@@ -192,115 +196,97 @@ angular.module('App').controller(
...
-                    ),
-                    err,
-                    ALERTER_ID,
-                  ),
-                );
-            default:
-              this.Alerter.success(
-                this.$translate.instant('domain_tab_OPERATION_update_success'),
-                ALERTER_ID,
+      for (const arg of this.args) {
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js:202]:
```suggestion
      this.args.forEach ({key, value, type, readOnly}) {
```
I think it's better to use foreach and destructured object

Comment (jvaumoroovh) [reply packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js:202]: with for...of, we can still use destructured objects.

In my opinion, for...of is more readable, and from a performance point of view, it's also better (not a big performance improvement, but an improvement nonetheless).

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js [hunk 2]
```diff
@@ -192,115 +196,97 @@ angular.module('App').controller(
...
+          this.displayAlert(
             this.$translate.instant('domain_tab_OPERATION_update_error'),
-            err,
+            ALERTER_ID,
+            error,
+          );
+        }
+      }
+
+      switch (this.todoOperation) {
```
Comment (antonymarion) [inline packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js:232]: Maybe replace switch case by dictionnary here

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-await-in-loop */
```
Comment (antonymarion) [inline packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js:1]: Maybe we can avoir disabling eslint here ?

File: packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html [hunk 4]
```diff
@@ -102,8 +102,7 @@
         data-compact
         data-disabled="!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate && $row.function !== 'DomainIncomingTransfer'"
         data-placement="end"
-        data-oui-tooltip="{{:: 'domain_operations_no_action' | translate }}"
-        data-oui-tooltip-enable="{{!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate}}"
+        data-oui-tooltip="{{ :: !$row.canCancel && !$row.canRelaunch && !$row.canAccelerate ? 'domain_operations_no_action' : '' | translate }}"
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html:105]: Is this working without any common bracket ?

Comment (sachinrameshn) [reply packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html:105]: data-oui-tooltip="{{ :: (!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate ? 'domain_operations_no_action' : '') | translate }}"

Comment (jvaumoroovh) [reply packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html:105]: It's working

image

File: packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js (hunk 3 above)
Comment (jvaumoroovh) [reply packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js]: no, we can't avoid disabling eslint here, I need to have await in a for loop to make my http calls sequential (or i have to add an eslint disable for each ligne that have await in the for loop

**Pull request merged**